doc/other_modules/jenny/language/markup.md

```diff
@@ -36,7 +39,7 @@ space after the tag will be removed from the resulting text. If this is undesire
 add an extra space after the markup tag:
 
 ```yarn
-Lorem ipsum dolor sit amet, [carramba/] consectetur adipiscing elit.
+Lorem ipsum dolor sit amet, [wave/] consectetur adipiscing elit.
```

Comment:
I used this documentation to understand what these mean and provide more understandable examples
https://docs.yarnspinner.dev/getting-started/writing-in-yarn/markup#self-closing-attributes
I found that the words used before made it quite hard to understand the purpose of tags